ansible-scylla-node/tasks/common.yml

```diff
@@ -114,23 +133,37 @@
   become: true
   when: skip_ntp is defined and skip_ntp|bool == false
 
+- name: check for node_exporter_install
+  stat:
+    path: /usr/sbin/node_exporter_install
+  register: node_exporter_install
+
 - name: node exporter setup
   shell: |
     node_exporter_install --force
-  when: install_type == 'online'
+  when: (install_type == 'online') and (node_exporter_install.stat.exists)
   become: true
   notify:
     - node_exporter start
-  ignore_errors: true
-  #TODO: stop ignoring errors when the node_exporter_install script fix is available in all actual versions, resp. use only for < 5.0 / 2022
+
+- name: check for node_exporter
+  stat:
+    path: /opt/scylladb/node_exporter/node_exporter
+  register: node_exporter
```

Comment:
This is not a check for a `node_exporter`. This checks for some ScyllaDB thingy. 

So at least the `name` part requires an adjustment.

Reply:
changed to : "check for old node_exporter installation"